docs/ground/departure_clearance.md

```diff
@@ -4,71 +4,70 @@ A departure clearance is a set of instructions given by air traffic control to a
-- **A**ltitude: Maximum altitude to which the aircraft is cleared. This may be lower than the altitude levels defined by the SID, if necessary for traffic spacing.
-- **F**requency: Frequency the aircraft must switch to when requested by the tower controller. Under ICAO procedures this is often omitted.
-- **T**ransponder: Mode 3 squawk code that the aircraft should set into its transponder.
+- **Clearance**: Point to which the clearance is valid. This may be a destination airfield, or perhaps an Operating Area, where the aircraft will be handed to tactical air control.
+- **Route**: Path that the aircraft will take to depart the airfield. May be vectors, or a predefined Standard Instrument Departure.
+- **Altitude** (Optional): Usually, the aircraft can just follow the altitude restrictions provided by the SID. However, an altitude restriction **MAY** be issued for separation or other reasons.
+- **Transponder**: Mode 3 squawk code that the aircraft should set into its transponder.
 
-!!!example IFR Clearance example
-Pilot: "Krasnodar Ground, Fastjet 345, [single/_number_] [aircraft type] at [apron/ramp name], request clearance to Sochi, IFR."
-Tower: "Fastjet 345, Krasnodar Ground, advise ready to copy IFR clearance."
-Pilot: "Ready to copy, Fastjet 345."
-Tower: "Fastjet 345, cleared to Sochi, RANIS6T departure, climb to flight level 170, squawk 1234."
-Pilot: "Cleared to Sochi, RANIS6T departure, climb to flight level 170, squawk 1234, Fastjet 345."
-Tower: "Fastjet 345, readback correct, report ready to taxi."
+!!! example "IFR Clearance example"
+    **Pilot**: "Krasnodar Ground, Fastjet 345, [single/_number_] [aircraft type] at [apron/ramp name], request clearance to Sochi, IFR."  
+    **Ground**: "Fastjet 345, Krasnodar Ground, advise ready to copy IFR clearance."  
+    **Pilot**: "Ready to copy, Fastjet 345."  
+    **Ground**: "Fastjet 345, cleared to Sochi, RANIS6T departure, squawk 1234."  
+    **Pilot**: "Cleared to Sochi, RANIS6T departure, squawk 1234, Fastjet 345."  
+    **Ground**: "Fastjet 345, readback correct, report ready to taxi."  
 
 A radar vector departure is as follows.  
 
-!!!example Departure via vectors
-[...]
-Tower: "Fastjet 345, cleared to Sochi, departure via radar vectors, climb to 5000 feet, [QNH1013 HPa/Altimeter 29.92], squawk 1234."
-Pilot: "cleared to Sochi, departure via radar vectors, climb to 5000 feet, [QNH1013 HPa/Altimeter 29.92], squawk 1234, Fastjet 345."
-Tower: "Fastjet 345, readback correct, report ready to taxi."
\ No newline at end of file
+!!! example "Departure via vectors"
+    [...]  
+    **Ground**: "Fastjet 345, cleared to Sochi, departure via radar vectors, climb to 5000 feet, [QNH1013 HPa/Altimeter 29.92], squawk 1234."  
+    **Pilot**: "cleared to Sochi, departure via radar vectors, climb to 5000 feet, [QNH1013 HPa/Altimeter 29.92], squawk 1234, Fastjet 345."  
```

Comment:
Wasn't sure whether this altitude instruction would be fine, or whether "departure via radar vectors" would be enough. Let me know if this needs to be changed.

Reply:
yeah, altitude would be given for radar vectors since there's nothing published, only thing would be that when the clearance is given at the hold it is given by tower rather than ground, it's basically to avoid coordination between ground and tower (and/or tower can't give the clearance at that time)